app/views/chapters/_form.html.erb

```diff
@@ -19,6 +19,7 @@
   <%= form.input :extra_question_1 %>
   <%= form.input :extra_question_2 %>
   <%= form.input :extra_question_3 %>
+  <%= form.input :default_email_text, label: "Customized Email Response", placeholder: "This will appear as the introduction in emails sent to applicants to your chapter." %>
```

Comment:
I feel like the placeholder should be the actual default text, and then maybe we can include the helper text below the box. That way it'll indicate that even if this is not customized for the chapter that a message will be included by default.

In addition, the actual label (and helper) should be in the localization file, not hard-coded into the view; we shouldn't include any language-specific content in any views.